src/components/Class/index.js

```diff
@@ -334,62 +364,102 @@ function Class({ classToEdit }) {
                         <select
                           className="create-class-select"
                           name={COURSE_ID}
                           value={formFieldsState[COURSE_ID]}
                           onChange={(e) => {
+                            onCourseChange(e.target.value);
                             setFormField(e.target.value, COURSE_ID);
                           }}
                           id="course_id"
                         >
-                          <option value="">
-                            Select a course from options below
-                          </option>
-                          {item.courses.map((course, index) => {
+                          <option value="">Select a course</option>
+                          {pathway.courses.map((course) => {
                             return (
-                              <option key={index} value={course.id}>
+                              <option key={course.id} value={course.id}>
                                 {course.name}
                               </option>
                             );
                           })}
                         </select>
-                      </>
+                      </React.Fragment>
                     );
                   }
                 })}
-              <label htmlFor="max_enrolment">Maximum Enrollments</label>
+              {formFieldsState[COURSE_ID] && exercisesForSelectedCourse && (
+                <>
+                  <label htmlFor="exercise_id">
+                    Select Exercise
+                    <span className="optional-field">(optional)</span>
+                  </label>
+                  <select
+                    className="create-class-select"
+                    name={EXERCISE_ID}
+                    value={formFieldsState[EXERCISE_ID]}
+                    onChange={(e) => {
+                      setFormField(e.target.value, EXERCISE_ID);
+                    }}
+                    id="exercise_id"
+                  >
+                    <option value="">Select an exercise</option>
+                    {(
+                      exercisesForSelectedCourse[formFieldsState[COURSE_ID]] ||
+                      []
+                    ).map((exercise) => {
+                      return (
+                        <option key={exercise.id} value={exercise.id}>
+                          {exercise.name}
+                        </option>
+                      );
+                    })}
+                  </select>
+                </>
+              )}
+              <label htmlFor={MAX_ENROLMENT}>
+                Maximum Enrollments
+                <span className="optional-field">(optional)</span>
+              </label>
               <input
                 className="input-field"
                 type="number"
-                name="max_enrolment"
-                id="max_enrolment"
-                min="1"
+                name={MAX_ENROLMENT}
```

Comment:
This is an `input` tag and not `select`.
So, I think the name should just be string, "max_enrolment"

Any reasoning behind keeping it as a value ?